Makefile

```
@@ -16,3 +16,6 @@ benchmark:
 checks:
 	go vet ${PKGS}
 	staticcheck ${PKGS}
+
```

How about adding the linter as the last step of this target?
```suggestion
	make lint

```

---

sounds good. I'll make the change